--- variety_lib/varietyconfig.py.orig	2016-02-28 07:33:12.192000000 +0000
+++ variety_lib/varietyconfig.py	2016-02-28 07:32:39.632000000 +0000
@@ -27,7 +27,7 @@
 
 # Where your project will look for your data (for instance, images and ui
 # files). By default, this is ../data, relative your trunk layout
-__variety_data_directory__ = '../data/'
+__variety_data_directory__ = '/usr/share/variety'
 __license__ = 'GPL-3'
 __version__ = '0.6.0'
 
